content/intro-to-storybook/angular/en/screen.md

```diff
@@ -334,7 +403,7 @@ We started from the bottom with `TaskComponent`, then progressed to `TaskListCom
 
 [**Component-Driven Development**](https://www.componentdriven.org/) allows you to gradually expand complexity as you move up the component hierarchy. Among the benefits are a more focused development process and increased coverage of all possible UI permutations. In short, CDD helps you build higher-quality and more complex user interfaces.
 
-We’re not done yet - the job doesn't end when the UI is built. We also need to ensure that it remains durable over time.
+We’re not done yet - the job doesn't end when the UI is built, and we also need to ensure that it remains durable over time.
```

Comment:
```suggestion
We’re not done yet - the job doesn't end when the UI is built. We also need to ensure that it remains durable over time.
```

Revert back, the grammar isn't quite right.

Reply:
Fixed with the latest commit for all frameworks.